### DISCUSSION TITLE
Feature: Add RTL support

### DISCUSSION THREAD

**Discussion by shoaibkhandev** [youzan/vant#11785] [Ideas]
It's good to have RTL support

**Comment by chenjiahan** [comment]
Yes, we should provide an option to support RTL.

For example, provide the `direction` prop in the ConfigProvider component, just like [antd](https://ant.design/components/config-provider).

**Reply by shoaibkhandev** [reply]
Exactly.

So let's suppose the IndexBar shows all the indexes on the right side but in the RTL it's suppose to show on the left side and also there is quite some places which uses e.g margin-left or padding-left so these properties also needed to be handled according to the direction

**Reply by chenjiahan** [reply]
Yes, the RLT will affect the layout of many components, so we need to carefully evaluate the impact on each component, refer to the design of iOS and Android to provide a complete solution. Hoping someone in the community can help us with this.

**Reply by shoaibkhandev** [reply]
Let me know when you guys pick this feature i will contribute it as well

**Reply by chenjiahan** [reply]
Great to hear that you are interested in contributing to the project! We will definitely keep you updated on the progress of the feature and let you know when it's ready for contributions.

**Comment by kaven85** [comment]
请问官方预计在什么版本提供支持呢？

**Comment by ZZF10369** [comment]
目前支持了吗
